src/generators/python/presets/Pydantic.ts

```diff
@@ -114,7 +117,7 @@ def unwrap_${dictionaryModel?.propertyName}(cls, data):
     .map((value) => `'${value.unconstrainedPropertyName}'`)
     .join(', ')}]
   ${dictionaryModel?.propertyName} = {}
-  for obj_key in list(data.keys()):
+  for obj_key in cls.list_keys(data):
```

Comment:
I still believe that this code will never called due to condition above
`if len(unknown_object_properties) == 0:`